src/components/Categorise/Mobile/CategoriseMobile.js

```diff
@@ -0,0 +1,39 @@
+import filter from "../../../icons/filter.png";
+import sort from "../../../icons/sort.png";
+import { Filter } from "./Filter";
+import { Sort } from "./Sort";
+import { useProductContext } from "../../../Context/ProductProvider";
+
+export const CategoriseMobile = () => {
+  const {
+    blockSortDisplay,
+    toggleSetSortDisplay,
+    toggleSetFilterDisplay,
+    blockFilterDisplay
+  } = useProductContext();
+
+  return (
+    <div>
+      <div className="categorise-footer flex-row ">
+        <div
+          className="categorise-footer-button border-right"
+          onClick={() => toggleSetSortDisplay()}
+        >
+          <img src={sort} className="filter-icon" />
+         <span> SORT </span>
+        </div>
+        <div
+          className="categorise-footer-button"
+          onClick={() => toggleSetFilterDisplay()}
+        >
+          <img src={filter} className="filter-icon" />
+         <span> FILTERS </span>
+        </div>
+      </div>
+
+      {blockSortDisplay ? <Sort /> : null}
+
+      {blockFilterDisplay ? <Filter /> : null}

```

Comment:
Here you can change the state variable name to something like displaySort and displayFilter, because blockSortDisplay seems to give a different meaning than what you are doing here.
Or maybe you can just change the logic to this:

```suggestion
      {blockSortDisplay ? null : <Sort />}

      {blockFilterDisplay ? null : <Filter />}

```